### Activity 04/10/19 - Lars & Sabrina

## Using NCBI resources

### Which chromosome is your protein located on? Note its location.

ALAD: Chromosome 9, location 113'386'312-113'401'338 (for GRCh38.p12; GCF_000001405.38)


### What information is contained in dbVar?

dbVar contains information about larger sequence variants, such as longer indels, copy number variations or tandem duplications.

For every variant, it lists the variant region ID (a unique identifier), the variant type, the number of variant calls and associated Study ID(s), the clinical assertion (pathogenicity), the exact location in the genome and the genes present in the affected region.

Each variant has its own page detailing additional information, including a genome view and documentation of how and where the information was gathered.

For the ALAD gene, there are currently 136 variants in dbVar.


### What information is in dbSNP?

dbSNP contains information about single nucleotide variants as well as short indels.

For every variant, it lists the variant name (a unique identifier), the variant type, the allele change(s), the exact location in the genome and affected gene, the clinical significance (pathogenicity), the validation methods used, and MAF and HGVS information.

Each variant has its own page detailing additional information, including the location as seen in the variation viewer, population frequency data, and documentation of how and where the information was gathered, including links to relevant publications.

For the ALAD gene, there are currently 4229 variants in dbSNP.


### What information is in ClinVar?

ClinVar aggregates information about genomic variation and its relationship to human health, providing information about diseases, phenotypes, genes and sequence variants.

For every variant, it lists its name - in a form that includes the variation and location as part of the name - the affected gene(s), the change in proteins resulting from the sequence alterations, the phenotypic condition(s) associated with the variant, the clinical significance (pathogenicity)the review status and accession info. Each variant has its own page detailing additional detailed information.

For the ALAD gene, there are currently 100 variants in ClinVar.


### How many variants do you have (when filtered)?

There are 6 pathogenic missense variants of ALAD which are in ClinVar (rs121912981, rs121912983, rs121912982, rs121912980, rs1800435, rs121912984). All of them are single nucleotide variants (dbSNP entries).


### What is the PDB ID of the wild type protein?

There are currently 3 entries for wild-type human ALAD:
* __5HNR__: X-ray diffraction crystal structure of octameric human native 5-aminolaevulinic acid dehydratase (released 2016-01-27)
* __5HNS__: X-ray diffraction crystal structure of octameric human recombinant 5-aminolaevulinic acid dehydratase, expressed in Escherichia coli BL21(DE3) (released 2016-01-27)
* __1E51__: X-ray diffraction crystal structure of octameric human native 5-aminolaevulinic acid dehydratase (released 2001-07-12)


### Variant table:

|Variant ID|Mutation|Mutation type|PDB ID|Location in protein|Close active or binding sites|Polyphen 2 score|Prediction about impact|
|----------|-------------|------|-------|---------|-------|--------------|-------|
|rs121912981|Val275Met|hydrophobic|_none_|Beta strand (273-277)|Substrate Schiff-base (252)|0.997 (sens. 0.41 spec. 0.98)|Probably damaging|
|rs121912983|Ala274Thr|polar|_none_|Beta strand (273-277)|Substrate Schiff-base (252)|1.000 (sens. 0.00 spec. 1.00)|Probably damaging|
|rs121912982|Arg240Trp|hydrophobic|_none_|Alpha helix (231-243)|Substrate Schiff-base (252), Zinc 2 (223)|1.000 (sens. 0.00 spec. 1.00)|Probably damaging|
|rs121912980|Gly133Arg|basic|_none_|Beta strand (131-133)|Zinc 1 catalytic (122,124,132), Zinc 2 (131)|1.000 (sens. 0.00 spec. 1.00)|Probably damaging|
|rs1800435|Lys59Asn|polar|_none_|Alpha helix (58-71)|none|0.000 (sens. 1.00 spec. 0.00)|benign|
    

### iCn3D structure figures:

The following files are provided:
* _5HNR_dimer.png_
* _5HNR_octamer.png_

Chains are colored light or dark blue.
Ions and chemical substrates are colored grey.
Locations of variant amino acids are colored and have their wild-type side chains displayed:
* Red: 275 Val → Met
* Orange: 274 Ala → Thr
* Yellow: 240 Arg → Trp
* Green: 133 Gly → Arg
* Pink: 59 Lys → Asn
