# Impact of genetic variation on three dimensional structure and function of proteins.
## 1. What is the biological question?
SNVs can have an important effect on the amino acid sequence (of course) and therefore, can also affect the 3D
structure of a protein leading to abnormal activity, aggregation, stability, binding, assembly and rearrangement.
Once a protein cannot fulfil is canonical purpose the cell’s equilibrium is no more and the cell can get out of the
cell cycle control (cancerous state). The goal of this study is to improve the understanding of the relationship
between point mutations and experimentally observed consequences in 3D.
## 2. What is the method?
2596 structures in PDB have been identified. For each one, the sequence of amino acid obtained from the
crystal differs from the UniProt sequence at one position (SNV). From the 2596 structures, the ones from
which the dbSNP mutation information didn’t match the informations from UniProt and 3D structure have
been excluded, as well as mappings of the same SNV to multiple PDB entries. It remained 374 structures. The
consequences of SNV have been reviewed by the literature, which gave the functional effect of changes in
gene products. The position, where the substitution is (Structural locations of the SNVs were categorized into
2 main groups: Surface and Buried and sub categorised in loop, alpha helix, beta strand) and consequences of
each SNVs have been extracted.
Classification of mutation has been made (if the SNV affects the activity, stability, binding, assembly or
rearrangement of the protein)
The frequency in which a SNV occurs in a population is determined by NHLBI Exome Sequencing Project
(ESP) Exome Variant server, and dbSNP.
## 3. What significant scientific contribution does the paper make?
They observed that 79% of the SNVs are on the surface of the protein, and 21% in the interior of the protein.
They could show that the SNVs in the Surface category have a higher likelihood of being found in Loop regions
than the Buried category. Changes caused by SNVs are more likely to be found in alpha helices and beta strands.
They could also determine that a broad range of possible effects could be attributed to a single residue change.
A single residue mutation can have multiple effects on the protein structure and function, which means that a
single SNV can affect more than one of the six categories.
The analysis of such a small dataset (only 374 data) makes the understanding of consequences of particular
genetic variations more challenging. As most of the softwares that try to predict the consequence of SNVs
classify SNVs as "disease causing" or "not disease causing", mutations often go undetected (as they do not
result in a disease phenotype). However, they can still affect the protein by changing the structure and function,
without causing any disease.
Finally, a comprehensive understanding of three-dimensional structure, dynamics, and biophysics of wild-type
and mutant proteins will be required to develop better tools that can make accurate predictions regarding the
consequences of genetic changes manifested at the atomic level in protein gene products.

# Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients.
## 1. What is the biological question?
The project’s purpose is to sequence as much tumors as possible to create a landscape of metastatic cancer
sequencing. The data will allow to do a molecular profiling to provide all patient with the opportunity to
receive a genetically matched therapy.
## 2. What is the method?
MSK-IMPACT is a custom hybridization capture–based assay encompassing all genes that are druggable by
approved therapies or are targets of experimental therapies being investigated in clinical trials at Memorial
Sloan Kettering Cancer Center (MSKCC), as well as frequently mutated genes in human cancer. MSKIMPACT
uses both blood and tumor sample to extract DNA and sequence it. Then MSK-IMPACT results are
compared to TCGA for the corresponding tumor subtypes.
## 3. What significant scientific contribution does the paper make?
The paper lay the foundation for personal medicine regarding tumor treatment. With the large data collected of
different tumor sequencing with their own mutations it is possible to influence treatment decisions and improve
outcomes for patient. The sharing of the data will also permit other searcher to refine the tumor analysis.
# H-RasK117R (PDB : 2QUZ)
We chose The Lys -> Arg (K117R) substitution in H-Ras (mutant PDB: 2QUZ).
The PDB page of 2QUZ shows only one entry.
X-ray diffraction of a protein crystal has been used to determine the protein structure.
Concerning the entry, its Uniprot ID is P01112. It contains one chain of 166 amino acids. Mg and GDP are the two
ligands of the protein.
![](prot.png)
